backend/cloudinary/cloudinary.go

```diff
@@ -66,18 +144,42 @@ type Fs struct {
 	root     string
 	opt      Options
 	features *fs.Features
-	pacer    *pacer.Pacer
+	srv      *rest.Client // the connection to the server
 	cld      *cloudinary.Cloudinary
 }
 
 // Object describes a cloudinary object
 type Object struct {
-	fs      *Fs
-	remote  string
-	size    int64
-	modTime time.Time
-	url     string
-	md5sum  string
+	fs           *Fs
+	remote       string
+	size         int64
+	modTime      time.Time
+	url          string
+	md5sum       string
+	publicID     string
+	resourceType string
+	deliveryType string
+}
+
+type updateOptions struct {
+	UpdateMode   bool
+	PublicID     string
+	ResourceType string
+	DeliveryType string
+}
+
+func (o *updateOptions) Header() (string, string) {
+	return "UpdateMode", fmt.Sprintf("%v", o.UpdateMode)
+}
+
+// Mandatory returns whether the option must be parsed or can be ignored
+func (o *updateOptions) Mandatory() bool {
+	return false
+}
+
+// String formats the option into human-readable form
+func (o *updateOptions) String() string {
+	return fmt.Sprintf("updateOptions(%v)", o.UpdateMode)
```

Comment:
Not sure if this is in use, if yes - please add other fields as well

Reply:
This is not in use

Reply:
Can we remove it then?